ui/src/app/applications/components/utils.tsx

```diff
@@ -245,10 +245,10 @@ export const ComparisonStatusIcon = ({
             break;
         case appModels.SyncStatuses.OutOfSync:
             const requiresPruning = resource && resource.requiresPruning;
-            className = requiresPruning ? 'fa fa-times-circle' : 'fa fa-arrow-alt-circle-up';
+            className = requiresPruning ? 'fa fa-trash' : 'fa fa-arrow-alt-circle-up';
```

Comment:
Does this change the icon everywhere, e.g. the filter on the left side?

Reply:
no, as requiresPruning is not set when generating the SyncFilter component, so only "OutOfSync" (with the arrow-up icon) shows up there.